test/z_cli_daemon_test.go

```diff
@@ -466,3 +466,19 @@ func (suite *PouchDaemonSuite) TestDaemonWithMultiRuntimes(c *check.C) {
 	c.Assert(err, check.NotNil)
 	dcfg2.KillDaemon()
 }
+
+func (suite *PouchDaemonSuite) TestUpdateDaemonWithLabels(c *check.C) {
+	cfg := daemon.NewConfig()
+	err := cfg.StartDaemon()
+	c.Assert(err, check.IsNil)
+
+	defer cfg.KillDaemon()
+
+	command.PouchRun("updatedaemon", "--label", "aaa=bbb").Assert(c, icmd.Success)
+
+	ret := command.PouchRun("info")
```

Comment:
Use:	ret = RunWithSpecifiedDaemon(&cfg, "version")

Or it will send request to the default pouchd instead of the testing one.